core/src/main/java/org/radargun/config/ConfigSchemaGenerator.java

```diff
@@ -63,7 +63,6 @@ protected void generate() {
       addAttribute(masterComplex, ATTR_PORT, intType, null, false);
 
       Element clustersChoice = createChoice(benchmarkSequence, 0, 1);
-      Element localComplex = createComplexElement(clustersChoice, ELEMENT_LOCAL, 0, 1);
```

Comment:
createComplexElement has a side-effect. You probably want just to remove the unused variable. The same for scenarioType.